### PR TITLE
feat(VETS-CPC-234): Visual changes on ui to implement file upload (resume and picture)

### PR DESCRIPTION
**Pull request by Chloedunwoody**
Screen Shot 2021-10-05 at 3 56 46 PM
Screen Shot 2021-10-05 at 3 57 12 PM
[https://champlainsaintlambert.atlassian.net/browse/CPC-234?atlOrigin=eyJpIjoiYWYxMjQzMmE0ODA5NDMyNzkwZmVhZTkxODlmMTU4ZTkiLCJwIjoiaiJ9](url)
Will not allow user to select unwanted file extensions.
Error message for wrong file type in awaiting stage. Waiting on validation branch.
Will need backend SQL to ensure file upload to the db

### CODE REVIEW

**Comment by Chloedunwoody** [comment]
Again will wait to do merging with a reviewer to ensure no code break.

File: vets-service/src/main/java/com/petclinic/vets/datalayer/Vet.java [hunk 1]
```diff
@@ -70,6 +70,9 @@
     @NotEmpty
     private String phoneNumber;
 
+    @Column(name = "profile_picture")
+    private String profilePicture;
+
```
Comment (SmoothWin) [inline vets-service/src/main/java/com/petclinic/vets/datalayer/Vet.java:75]: Isn't this supposed to be handled by another story and not yours?

**Review by SmoothWin — CHANGES_REQUESTED** [review]
Should all those image field changes for the vet implementation be implemented by another story and not your story:
Here is the story that handles the change to the vet field for images:
[](https://champlainsaintlambert.atlassian.net/jira/software/c/projects/CPC/boards/13?modal=detail&selectedIssue=CPC-237&search=vets)

File: vets-service/src/main/java/com/petclinic/vets/datalayer/Vet.java (hunk 1 above)
Comment (Chloedunwoody) [reply vets-service/src/main/java/com/petclinic/vets/datalayer/Vet.java]: Changes have now been pushed.

**Review by SmoothWin — APPROVED** [review]
Changes are satisfactory.

**Pull request merged**